conf/application.conf

```diff
@@ -16,8 +16,8 @@
 # ~~~~~
 
 appName=lisa-api
-appUrl = "http://lisa-api.service"
-//appUrl="http://localhost:9000"
+//appUrl = "http://lisa-api.service"
+appUrl="http://localhost:9000"
```

Comment:
Is this correct?

Reply:
yeah , to have it pointed to localhost so we can use for documentation preview